staging/src/k8s.io/component-base/config/testing/roundtrip.go

```diff
@@ -56,6 +56,9 @@ func GetRoundtripTestCases(scheme *runtime.Scheme, disallowMarshalGroupVersions
 				continue // Don't try to deserialize the internal version
 			}
 			for _, vout := range versions {
+				if vout == runtime.APIVersionInternal {
```

Comment:
Will this result in no test cases as currently written? 

Round-trip tests should be testing that for a conversion external->internal->external, the external is the same as how it started. This was to some degree tested before, since we tested external->internal and internal->external segments separately based on the serialized content, and internal values were the same in each piecemeal case, but now I think we either have to change the test to do external->internal->external altogether instead of piecemeal, or we have to represent the expected internal value in a runtime structure to avoid serialized internal yaml.

@liggitt WDYT?

Reply:
This should also not be attempting direct conversions between external types, but it looks like that could happen with multiple external versions?

Reply:
> now I think we either have to change the test to do external->internal->external altogether instead of piecemeal, or we have to represent the expected internal value in a runtime structure to avoid serialized internal yaml.

For configuration, we've traditionally tested that given serialized config bytes produced a specific in-memory representation, but either approach could work